libindy_vdr/src/ffi/ledger.rs

```diff
@@ -602,3 +604,196 @@ pub extern "C" fn indy_vdr_build_get_rich_schema_object_by_metadata_request(
         Ok(ErrorCode::Success)
     }
 }
+
+#[no_mangle]
+pub extern "C" fn indy_vdr_build_node_request(
+    identifier: FfiStr,
+    dest: FfiStr,
+    data: FfiStr,
+    handle_p: *mut RequestHandle,
+) -> ErrorCode {
+    catch_err! {
+        trace!("Build NODE request");
+        check_useful_c_ptr!(handle_p);
+        let builder = get_request_builder()?;
+        let identifier = DidValue::from_str(identifier.as_str())?;
+        let dest = DidValue::from_str(dest.as_str())?;
+        let node_data = serde_json::from_str::<NodeOperationData>(data.as_str())
+            .with_input_err("Error deserializing NodeData")?;
+        let req = builder.build_node_request(&identifier, &dest, node_data)?;
+        let handle = add_request(req)?;
+        unsafe {
+            *handle_p = handle;
+        }
+        Ok(ErrorCode::Success)
+    }
+}
+
+#[no_mangle]
+pub extern "C" fn indy_vdr_build_pool_config(
+    identifier: FfiStr,
+    writes: bool,
```

Comment:
Can we use i8/c_int8 here? I'm not sure we can rely on both Rust's bool and c_bool being the same size.